weblab/core/recipes.py

```diff
@@ -0,0 +1,10 @@
+from model_mommy.recipe import Recipe, seq
+
+
+user = Recipe('accounts.User', institution='UCL')
+
+model = Recipe('ModelEntity', visibility='public', entity_type='model', name=seq('mymodel'))
```

Comment:
Is this to make our testing code easier to write? Or are there performance benefits involved?

Reply:
Not (just) being grumpy here, but a bit concerned about adding dependencies (on what looks to be a 2 person project) for simple things like object creation! Every new dependency has the potential to increase the learning curve for new developers and cause issues for installation / testing / cross-platform availability...

Reply:
@MichaelClerx It makes testing code easier to write and clearer to read (prevents repetition of setting up irrelevant but required fields on test objects). The dependency is contained entirely within the pip dependencies. I feel like it's useful to have more concise tests, but interested in what others think.

Reply:
It's been used (and useful) on the Dashboard project too, and I agree about the benefits of clearer test code. We'd probably end up writing such helpers ourselves if we didn't use it...